license-maven-plugin-fs/src/test/java/com/mycila/maven/plugin/license/fs/CopyrightRangeProviderTest.java

```diff
@@ -84,7 +84,7 @@ private static Document newDocument(String relativePath) {
 
   @BeforeAll
   static void beforeClass() throws IOException {
-    fsRepoRoot = Path.of(tempFolder.toPath() + File.separator + "fs-test-repo");
+    fsRepoRoot = Path.of(tempFolder + File.separator + "fs-test-repo");
```

Comment:
In theory this should be: `Path.of(tempFolder, "fs-test-repo");`

should this (and the others) be changed also in this PR ?

Reply:
ah! just saw tempFolder is a Path already... so: tempFolder.resolve(...) ?

Reply:
makes sense, yes will change that.

Reply:
Used toString(), haven't looked at resolve yet.  There were others. I can look at that if this still looks to need some more work.

Reply:
Yeah it does a lot of operations for nothing ;-)

fsRepoRoot = tempFolder.resove("fs-test-repo")

Reply:
pushed updates